internal/corazarules/rule_match.go

```diff
@@ -125,25 +125,23 @@ func (mr MatchedRule) writeDetails(log *strings.Builder, matchData types.MatchDa
 		mr.ServerIPAddress_, mr.URI_, mr.TransactionID_))
 }
 
-func (mr MatchedRule) matchData(matchData types.MatchData) string {
-	log := &strings.Builder{}
-	for _, matchData := range mr.MatchedDatas_ {
-		v := matchData.Variable().Name()
-		if matchData.Key() != "" {
-			v += fmt.Sprintf(":%s", matchData.Key())
-		}
-		value := matchData.Value()
-		if len(value) > 200 {
-			value = value[:200]
-		}
-		if mr.Rule_.Operator() != "" {
-			log.WriteString(fmt.Sprintf("Matched \"Operator %s matched %s at %s.",
-				"", value, v))
-		} else {
-			log.WriteString("Matched.\"")
-		}
+func (mr MatchedRule) matchData(matchData types.MatchData, log *strings.Builder) {
+	value := matchData.Value()
```

Comment:
Thanks! It looks like the `Operator() != ""` case is removed, just checking if that's intended. Actually, I do sort of suspect matchdata is always supposed to have an operator with it

Reply:
Cc @jptosso

Reply:
sorry,it seems the operator could be empty
https://github.com/corazawaf/coraza/blob/21dfb41e12d2dcbb0fb736b633aa03c546a6d90a/internal/seclang/rule_parser.go#L354C19-L380